src/main/java/pl/ateam/disasteralerts/user/UserService.java

```diff
@@ -90,4 +97,13 @@ public long userCount() {
         return userRepository.count();
     }
 
+    public void updateUserEntity(UserUpdateDTO userUpdateDto, UUID userId) {
+        User user = findById(userId);
+        user.setLocation(userUpdateDto.location());
+        user.setEmail(userUpdateDto.email());
+        user.setUsername(userUpdateDto.username());
+        user.setPhoneNumber(userUpdateDto.phoneNumber());
+
+        userRepository.save(user);
+    }
```

Comment:
Ta metoda mogła by być oznaczona jako @Transactional co zwiększa gwarancję spójności danych w db, a jeżeli się na to zdecydujesz to linia userRepository.save(user); jest zbędna bo zadziała mechanizm 'dirty checking'